components/Home/sidebar/sidebar_items/FeaturedDocSidebarItem.tsx

```diff
@@ -0,0 +1,85 @@
+import { css, StyleSheet } from "aphrodite";
+import { FEDocType } from "~/config/utils/getUnifiedDocType";
+import { UnifiedDocument } from "~/config/types/root_types";
+import { ReactElement } from "react";
+import AuthorFacePile from "~/components/shared/AuthorFacePile";
+import colors from "~/config/themes/colors";
+
+export default function FeaturedDocSidebarItem({
+  createdBy,
+  document,
+  documentType,
+  id,
+}: UnifiedDocument): ReactElement {
+  const { id: relatedDocID, slug, title } = document ?? {};
+  return (
+    <div className={css(styles.featuredDocSidebarItemContainer)}>
+      {/* NOTE: href is subject to change */}
+      <a
+        style={{ textDecoration: "none" }}
+        href={`/${
+          documentType === "question" ? "post" : documentType
```

Comment:
Perhaps you can use `getUrlToUniDoc`

Reply:
some bounty types dont come in with unified document. This is good as is for now. 
But I'll have to see if i can add better link generator function later